modules/mailer/src/main/AutomaticEmail.scala

```diff
@@ -174,6 +177,51 @@ To make a new donation, head to $baseUrl/patron"""
       }
     }
 
+  def dailyCorrespondenceNotice(
+      userId: User.ID,
+      opponents: List[lila.hub.actorApi.mailer.CorrespondenceOpponent]
```

Comment:
Using an `actorApi` class directly, not sure if it's ok since I didn't come across it elsewhere in the codebase